[2024-08-24T20:12:53.479+0000] {processor.py:186} INFO - Started process (PID=747) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:12:53.481+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:12:53.485+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:12:53.483+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:12:53.492+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:12:53.489+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 11
    
    ^
IndentationError: expected an indented block after 'with' statement on line 6
[2024-08-24T20:12:53.493+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:12:53.549+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.084 seconds
[2024-08-24T20:12:58.562+0000] {processor.py:186} INFO - Started process (PID=748) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:12:58.564+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:12:58.567+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:12:58.566+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:12:58.582+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:12:58.574+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 11, in <module>
    t1 = HEllo
         ^^^^^
NameError: name 'HEllo' is not defined
[2024-08-24T20:12:58.587+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:12:58.650+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.104 seconds
[2024-08-24T20:13:02.679+0000] {processor.py:186} INFO - Started process (PID=749) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:02.680+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:13:02.683+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:02.682+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:02.705+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:02.689+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 11, in <module>
    t1 = HelloOperator()
         ^^^^^^^^^^^^^
NameError: name 'HelloOperator' is not defined. Did you mean: 'PythonOperator'?
[2024-08-24T20:13:02.716+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:02.742+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.076 seconds
[2024-08-24T20:13:06.763+0000] {processor.py:186} INFO - Started process (PID=750) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:06.765+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:13:06.767+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:06.767+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:06.792+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:06.774+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 10, in <module>
    t1 = HelloOperator()
         ^^^^^^^^^^^^^
NameError: name 'HelloOperator' is not defined
[2024-08-24T20:13:06.808+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:06.850+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.099 seconds
[2024-08-24T20:13:19.109+0000] {processor.py:186} INFO - Started process (PID=759) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:19.209+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:13:19.214+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:19.213+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:19.369+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:19.303+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 10, in <module>
    t1 = HelloOperator()
         ^^^^^^^^^^^^^
NameError: name 'HelloOperator' is not defined
[2024-08-24T20:13:19.401+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:19.470+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.375 seconds
[2024-08-24T20:13:21.304+0000] {processor.py:186} INFO - Started process (PID=760) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:21.311+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:13:21.316+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:21.315+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:21.358+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:21.331+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 10, in <module>
    t1 = HelloOperator()
         ^^^^^^^^^^^^^
NameError: name 'HelloOperator' is not defined
[2024-08-24T20:13:21.384+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:21.432+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.149 seconds
[2024-08-24T20:13:47.700+0000] {processor.py:186} INFO - Started process (PID=769) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:47.875+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:13:47.878+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:47.877+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:47.937+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:47.919+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 10, in <module>
    t1 = HelloOperator(task_id="")
         ^^^^^^^^^^^^^
NameError: name 'HelloOperator' is not defined
[2024-08-24T20:13:47.951+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:47.979+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.289 seconds
[2024-08-24T20:13:51.887+0000] {processor.py:186} INFO - Started process (PID=770) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:51.888+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:13:51.891+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:51.890+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:51.918+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:51.898+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 10, in <module>
    t1 = HelloOperator(task_id="hello")
         ^^^^^^^^^^^^^
NameError: name 'HelloOperator' is not defined
[2024-08-24T20:13:51.938+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:51.989+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.118 seconds
[2024-08-24T20:13:53.944+0000] {processor.py:186} INFO - Started process (PID=771) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:53.948+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:13:53.951+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:53.950+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:53.986+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:53.959+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 10, in <module>
    t1 = HelloOperator(task_id="hello",
         ^^^^^^^^^^^^^
NameError: name 'HelloOperator' is not defined
[2024-08-24T20:13:54.005+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:13:54.063+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.160 seconds
[2024-08-24T20:14:06.172+0000] {processor.py:186} INFO - Started process (PID=772) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:14:06.173+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:14:06.177+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:14:06.176+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:14:06.201+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:14:06.182+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 10, in <module>
    t1 = HelloOperator(task_id="hello",
         ^^^^^^^^^^^^^
NameError: name 'HelloOperator' is not defined
[2024-08-24T20:14:06.214+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:14:06.254+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.098 seconds
[2024-08-24T20:14:15.623+0000] {processor.py:186} INFO - Started process (PID=781) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:14:15.625+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:14:15.628+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:14:15.627+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:14:15.671+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:14:15.635+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 10, in <module>
    t1 = HelloOperator(task_id="hello",
         ^^^^^^^^^^^^^
NameError: name 'HelloOperator' is not defined
[2024-08-24T20:14:15.706+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:14:15.756+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.148 seconds
[2024-08-24T20:14:16.701+0000] {processor.py:186} INFO - Started process (PID=782) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:14:16.703+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:14:16.706+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:14:16.706+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:14:16.736+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:14:16.715+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 10, in <module>
    t1 = HelloOperator(task_id="hello",
         ^^^^^^^^^^^^^
NameError: name 'HelloOperator' is not defined
[2024-08-24T20:14:16.756+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:14:16.810+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.126 seconds
[2024-08-24T20:14:47.753+0000] {processor.py:186} INFO - Started process (PID=790) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:14:47.768+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:14:47.772+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:14:47.771+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:14:47.798+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:14:47.779+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 10, in <module>
    t1 = HelloOperator(task_id="hello",
         ^^^^^^^^^^^^^
NameError: name 'HelloOperator' is not defined
[2024-08-24T20:14:47.812+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:14:47.858+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.124 seconds
[2024-08-24T20:15:18.356+0000] {processor.py:186} INFO - Started process (PID=798) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:15:18.369+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:15:18.371+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:15:18.371+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:15:18.389+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:15:18.376+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 10, in <module>
    t1 = HelloOperator(task_id="hello",
         ^^^^^^^^^^^^^
NameError: name 'HelloOperator' is not defined
[2024-08-24T20:15:18.407+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:15:18.438+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.097 seconds
[2024-08-24T20:15:48.982+0000] {processor.py:186} INFO - Started process (PID=806) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:15:48.994+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:15:48.998+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:15:48.997+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:15:49.043+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:15:49.003+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 10, in <module>
    t1 = HelloOperator(task_id="hello",
         ^^^^^^^^^^^^^
NameError: name 'HelloOperator' is not defined
[2024-08-24T20:15:49.066+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:15:49.149+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.181 seconds
[2024-08-24T20:16:19.688+0000] {processor.py:186} INFO - Started process (PID=814) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:16:19.708+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:16:19.714+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:16:19.713+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:16:19.746+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:16:19.721+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 10, in <module>
    t1 = HelloOperator(task_id="hello",
         ^^^^^^^^^^^^^
NameError: name 'HelloOperator' is not defined
[2024-08-24T20:16:19.767+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:16:19.828+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.163 seconds
[2024-08-24T20:16:50.244+0000] {processor.py:186} INFO - Started process (PID=822) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:16:50.254+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:16:50.258+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:16:50.257+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:16:50.294+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:16:50.264+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 10, in <module>
    t1 = HelloOperator(task_id="hello",
         ^^^^^^^^^^^^^
NameError: name 'HelloOperator' is not defined
[2024-08-24T20:16:50.315+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:16:50.356+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.130 seconds
[2024-08-24T20:17:20.731+0000] {processor.py:186} INFO - Started process (PID=831) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:17:20.733+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:17:20.735+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:17:20.735+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:17:20.750+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:17:20.739+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 10, in <module>
    t1 = HelloOperator(task_id="hello",
         ^^^^^^^^^^^^^
NameError: name 'HelloOperator' is not defined
[2024-08-24T20:17:20.764+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:17:20.801+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.083 seconds
[2024-08-24T20:17:51.569+0000] {processor.py:186} INFO - Started process (PID=840) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:17:51.669+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:17:51.672+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:17:51.671+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:17:51.690+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:17:51.678+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 10, in <module>
    t1 = HelloOperator(task_id="hello",
         ^^^^^^^^^^^^^
NameError: name 'HelloOperator' is not defined
[2024-08-24T20:17:51.704+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:17:51.741+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.187 seconds
[2024-08-24T20:18:04.763+0000] {processor.py:186} INFO - Started process (PID=841) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:18:04.766+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:18:04.770+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:18:04.768+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:18:04.784+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:18:04.775+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/3-customoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/3-customoperator.py", line 3, in <module>
    frim
NameError: name 'frim' is not defined
[2024-08-24T20:18:04.788+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:18:04.826+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.082 seconds
[2024-08-24T20:18:29.084+0000] {processor.py:186} INFO - Started process (PID=850) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:18:29.151+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:18:29.153+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:18:29.153+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:18:29.174+0000] {processor.py:925} INFO - DAG(s) 'customoperator' retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:18:29.488+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:18:29.486+0000] {override.py:1858} INFO - Created Permission View: can edit on DAG:customoperator
[2024-08-24T20:18:29.517+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:18:29.516+0000] {override.py:1858} INFO - Created Permission View: can read on DAG:customoperator
[2024-08-24T20:18:29.532+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:18:29.531+0000] {override.py:1858} INFO - Created Permission View: can delete on DAG:customoperator
[2024-08-24T20:18:29.534+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:18:29.533+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:18:29.571+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:18:29.570+0000] {dag.py:3234} INFO - Creating ORM DAG for customoperator
[2024-08-24T20:18:29.591+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:18:29.590+0000] {dag.py:4138} INFO - Setting next_dagrun for customoperator to 2024-08-01 00:00:00+00:00, run_after=2024-08-01 00:00:00+00:00
[2024-08-24T20:18:29.626+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.555 seconds
[2024-08-24T20:18:59.753+0000] {processor.py:186} INFO - Started process (PID=859) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:18:59.754+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:18:59.762+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:18:59.761+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:18:59.782+0000] {processor.py:925} INFO - DAG(s) 'customoperator' retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:18:59.848+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:18:59.848+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:18:59.934+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:18:59.934+0000] {dag.py:4138} INFO - Setting next_dagrun for customoperator to None, run_after=None
[2024-08-24T20:18:59.986+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.264 seconds
[2024-08-24T20:19:30.356+0000] {processor.py:186} INFO - Started process (PID=868) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:19:30.379+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:19:30.383+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:19:30.382+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:19:30.438+0000] {processor.py:925} INFO - DAG(s) 'customoperator' retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:19:30.622+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:19:30.622+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:19:30.744+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:19:30.743+0000] {dag.py:4138} INFO - Setting next_dagrun for customoperator to None, run_after=None
[2024-08-24T20:19:30.806+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.478 seconds
[2024-08-24T20:20:01.003+0000] {processor.py:186} INFO - Started process (PID=877) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:20:01.004+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:20:01.008+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:20:01.007+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:20:01.029+0000] {processor.py:925} INFO - DAG(s) 'customoperator' retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:20:01.086+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:20:01.086+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:20:01.150+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:20:01.150+0000] {dag.py:4138} INFO - Setting next_dagrun for customoperator to None, run_after=None
[2024-08-24T20:20:01.187+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.201 seconds
[2024-08-24T20:20:31.597+0000] {processor.py:186} INFO - Started process (PID=886) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:20:31.617+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:20:31.619+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:20:31.619+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:20:31.651+0000] {processor.py:925} INFO - DAG(s) 'customoperator' retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:20:31.720+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:20:31.720+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:20:31.775+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:20:31.775+0000] {dag.py:4138} INFO - Setting next_dagrun for customoperator to None, run_after=None
[2024-08-24T20:20:31.814+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.230 seconds
[2024-08-24T20:21:02.121+0000] {processor.py:186} INFO - Started process (PID=896) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:21:02.137+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:21:02.139+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:21:02.139+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:21:02.162+0000] {processor.py:925} INFO - DAG(s) 'customoperator' retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:21:02.229+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:21:02.228+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:21:02.287+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:21:02.287+0000] {dag.py:4138} INFO - Setting next_dagrun for customoperator to None, run_after=None
[2024-08-24T20:21:02.319+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.213 seconds
[2024-08-24T20:21:32.596+0000] {processor.py:186} INFO - Started process (PID=905) to work on /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:21:32.612+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/3-customoperator.py for tasks to queue
[2024-08-24T20:21:32.614+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:21:32.613+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:21:32.632+0000] {processor.py:925} INFO - DAG(s) 'customoperator' retrieved from /opt/airflow/dags/3-customoperator.py
[2024-08-24T20:21:32.674+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:21:32.674+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:21:32.718+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:21:32.718+0000] {dag.py:4138} INFO - Setting next_dagrun for customoperator to None, run_after=None
[2024-08-24T20:21:32.749+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/3-customoperator.py took 0.164 seconds
